aepsych/server/sockets.py

```diff
@@ -7,14 +7,17 @@
 
 import json
 import logging
+import select
 import socket
 import sys
+import time
 
 import aepsych.utils_logging as utils_logging
 import numpy as np
 import zmq
 
 logger = utils_logging.getLogger(logging.INFO)
+BAD_REQUEST = "bad request"
```

Comment:
We shouldn't define the same constant in multiple places. Define it in one place and import it wherever you need it.

Reply:
Done. Defined in sockets, imported everywhere else.